tests/e2e/kepler_internal_test.go

```diff
@@ -54,19 +73,114 @@ func TestKeplerInternal_Reconciliation(t *testing.T) {
 	assert.Equal(t, 1, len(containers))
 	assert.Equal(t, 1, len(containers[0].Ports))
 	assert.EqualValues(t, 9108, containers[0].Ports[0].ContainerPort)
+	// test expected status
+	testKeplerInternalStatus(t, f, ki)
+}
 
-	// the status will be updated
-	ki = f.WaitUntilInternalCondition(ki.Name, v1alpha1.Reconciled, v1alpha1.ConditionTrue)
-	assert.Equal(t, []corev1.Toleration{{Operator: "Exists"}}, ki.Spec.Exporter.Deployment.Tolerations)
+func TestKeplerInternal_WithEstimator(t *testing.T) {
+	f := test.NewFramework(t)
+	name := "e2e-kepler-internal-est"
```

Comment:
```suggestion
	name := "e2e-ki-modelserver-test"
```

Reply:
Why we need the suffix "-test"? If you think we should make kepler-internal in short, how's about `e2e-ki-estimator`, `e2e-ki-modelserver`, `e2e-ki-estimator-modelserver` for the newly-introduced case? (btw, it will have a different format from the basic case which is `e2e-kepler-internal`).